### PR TITLE
check for error code when LMSFinish/Terminate fails

### PR DESCRIPTION
**Pull request by moloko**
fixes https://github.com/adaptlearning/adapt_framework/issues/3048

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/scorm/wrapper.js [hunk 1]
```diff
@@ -336,7 +336,12 @@ define([
       this.lmsConnected = false;
 
       if (!this.scorm.quit()) {
-        this.handleError(new ScormError(CLIENT_COULD_NOT_FINISH));
+        const errorCode = this.scorm.debug.getCode();
+        this.handleError(new ScormError(CLIENT_COULD_NOT_FINISH, {
+          errorCode,
```
Comment (oliverfoster) [inline js/scorm/wrapper.js:341]: Where does this error info come out?

[Message definition](https://github.com/adaptlearning/adapt-contrib-spoor/blob/7173b1269cc84a4ebe10b8be6c8b6398501da94b/js/scorm/error.js#L30)
[Error handler](https://github.com/adaptlearning/adapt-contrib-spoor/blob/7173b1269cc84a4ebe10b8be6c8b6398501da94b/js/scorm/wrapper.js#L498-L532)

Comment (moloko) [reply js/scorm/wrapper.js:341]: Ha! Good point. I'll sort...

**Review by eleanor-heath — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**